src/index.html

```diff
@@ -11,7 +11,122 @@
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="text"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="number"
+            min="10"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="checkbox">
+        </label>
+      </fieldset> <!--Personal info-->
+
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input type="password" name="password">
+        </label>
+      </fieldset> <!--Registration-->
+
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?
+          <input type="radio" name="answer"> Yes
+          <input type="radio" name="answer"> No
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="time">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="brand" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            min="0"
+            max="10"
+            value="0"
+          >
+        </label>
+      </fieldset> <!--fact about you-->
+
+      <fieldset class="form-fieldset">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea name="comments" autocomplete="off"></textarea>
+        </label>
+
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommendation">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset> <!--Additional info-->
+
+      <button type="submit" name="Submit">Submit</button>
```

Comment:
[checklsit](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 4th item
```suggestion
      <button type="submit" name="submit">Submit</button>
```